src/main/pages/che-7/hands-on-guidance/proc_adding-support-for-a-new-language.adoc

```diff
@@ -1,27 +1,38 @@
 [id="adding-support-for-a-new-language_{context}"]
 = Adding support for a new language
 
-This paragraph is the procedure module introduction: a short description of the procedure.
+This section describes how to add support for a new language into Eclipse Che.
```

Comment:
I would add a quick explanation of what adding language support consists of: it's a VS Code plugin that uses the languageserver-node library to hook into the VS Code API.